CONTRIBUTING.md

```diff
@@ -271,6 +272,21 @@ If you have made more than one commit, then you will need to _squash_ your commi
 
 To do this, see [Squashing Your Commits](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231).
 
+### Commit message
```

Comment:
Also, here. Please change to `Commit Message` instead of `Commit message`.

Reply:
Changes done.